docs/source/azure/policy/resources/publicip.rst

```diff
@@ -19,38 +19,12 @@ Actions
 Example Policies
 ----------------
 
-This set of policies will mark all public IP addresses for deletion in 7 days that have 'test' in name (ignore case),
-and then perform the delete operation on those ready for deletion.
+This policy will find all public IP addresses under DDoS attack over the last 72 hours
```

Comment:
I don't quite understand this example. Can we provide more details on why this indicates a DDoS attack?

Reply:
Github is showing the diff weird. If you expand the 3 lines it collapsed you'll see this was a previous example and the metric it 'IfUnderDDosAttack'